### PR TITLE
Integration & Unit tests

### PR DESCRIPTION
**Pull request by ToStefan**

### CODE REVIEW

File: src/test/java/hybrid/it/internship/library/integration/IntegrationControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                .andExpect(status().isOk());
+    }
+
+    @WithMockUser(value = "s")
+    @Test
+    public void testCreateRent() throws Exception {
+
+        LocalDateTime ldt = LocalDateTime.now();
+        RentDTO rentDto = RentDTO.builder()
+                .userId((long) 1)
```
Comment (babicmilan) [inline src/test/java/hybrid/it/internship/library/integration/IntegrationControllerTest.java]: Don't cast an integer to a long, simply write a long number as 1L

File: src/test/java/hybrid/it/internship/library/integration/IntegrationControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    private MockMvc mockMvc;
+
+    @Before
+    public void setUp() {
+        mockMvc = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
+    }
+
+    @WithMockUser(value = "s")
+    @Test
+    public void testGetBookById() throws Exception {
```
Comment (babicmilan) [inline src/test/java/hybrid/it/internship/library/integration/IntegrationControllerTest.java:50]: You could test getBookById for an id that isn't known in the DB. No need to do it, just a remark.

File: src/test/java/hybrid/it/internship/library/unit/BookServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+
+@RunWith(MockitoJUnitRunner.class)
+@SpringBootTest
+public class BookServiceTest {
+
+    private static PageDTO pageDto = new PageDTO(1, 1);
```
Comment (babicmilan) [inline src/test/java/hybrid/it/internship/library/unit/BookServiceTest.java]: Any special reason this is static?

Comment (ToStefan) [reply src/test/java/hybrid/it/internship/library/unit/BookServiceTest.java]: no

**Pull request merged**